### PR TITLE
Added airbnb eslint config

### PR DESCRIPTION
**Pull request by tchcxp**
Install all Eslint dependencies at runtime.

### CODE REVIEW

File: generators/eslint/index.js [hunk 1]
```diff
@@ -16,17 +16,15 @@ module.exports = class extends Generator {
...
-      },
       'lint-staged': rootPkg['lint-staged'],
-      eslintConfig: rootPkg.eslintConfig,
+      eslintConfig: {
+        extends: ['airbnb'],
+        env: {
+          mocha: true,
+          node: true
+        },
+        rules: {}
```
Comment (brettgullan) [inline generators/eslint/index.js]: Do we want to add the rules we've used in Stamp (at least) that customize/override some of the Airbnb defaults?

Comment (brettgullan) [reply generators/eslint/index.js]:
```
{
  "extends": "airbnb",
  "rules": {
    "arrow-body-style": "off",
    "arrow-parens": ["error", "always"],
    "comma-dangle": "off",
    "func-names": "off",
    "global-require": "off",
    "max-len": ["error", 120],
    "no-console": "off",
    "no-underscore-dangle": "off",
    "no-use-before-define": "off",
    "semi": ["error", "never"],
    "space-before-function-paren": ["error", "never"],
    "import/no-extraneous-dependencies": "off"
  },
  "env": {
    "node": true,
    "mocha": true
  }
}
```

Comment (tchcxp) [reply generators/eslint/index.js]: Updated. Thanks,

Comment (brettgullan) [reply generators/eslint/index.js]: OK. The rest LGTM.

**Pull request merged**